packages/superset-ui-chart-controls/src/components/ControlHeader.tsx

```diff
@@ -57,7 +58,16 @@ export default function ControlHeader({
   const renderOptionalIcons = () => {
     if (hovered) {
       return (
-        <span>
+        <span
+          css={(theme: any) => css`
```

Comment:
```suggestion
          css={(theme: SupersetTheme) => css`
```

Reply:
Is using `css` attribute preferable to creating a separate `styled.span`? I'm asking for future reference 🙂